packages/react-ui-core/src/Dropdown/Dropdown.js

```diff
@@ -53,6 +53,25 @@ export default class Dropdown extends Component {
     return parseArgs(anchorField, AnchorButton)
   }
 
+  renderChildren() {
+    const {
+      children,
+    } = this.props
+
+    if (Array.isArray(children)) {
+      const clonedChildren = React.Children.map(children, child =>
+        cloneElement(child, {
+          toggleVisibilty: this.toggleVisibilty,
+        })
+      )
+      return clonedChildren
+    }
+
+    return cloneElement(children, {
+      toggleVisibilty: this.toggleVisibilty,
```

Comment:
If you're passing the props in two possible ways, maybe it would be better to set this as a const at the top and use it?
```
const props = { toggleVisibilty: this.toggleVisibilty }


if (Array.isArray(children)) {
  return React.Children.map(children, child => cloneElement(child, props))
}

return cloneElement(children, props)